tools/FindPythonLibsNew.cmake

```diff
@@ -151,9 +151,13 @@ if(NOT _PYTHON_SUCCESS MATCHES 0)
   return()
 endif()
 
+option(
+  PYBIND11_PYTHONLIBS_OVERRWRITE
+  "Overwrite cached values read from Python library (classic search). Turn off if cross-compiling and manually setting these values."
+  ON)
 # Can manually set values when cross-compiling
 macro(_PYBIND11_GET_IF_UNDEF lst index name)
-  if(NOT DEFINED "${name}")
+  if(PYBIND11_PYTHONLIBS_OVERRWRITE OR NOT DEFINED "${name}")
```

Comment:
Looks like one `R` to many: `OVERRWRITE`

I don't understand the high level enough to say something meaningful. What I see looks fine with my very superficial understanding.

Reply:
```suggestion
  if(PYBIND11_PYTHONLIBS_OVERWRITE OR NOT DEFINED "${name}")
```